xpdacq/tests/test_beamtimeSetup.py

```diff
@@ -94,9 +94,9 @@ def test_start_beamtime(self):
         self.assertEqual(self.experimenters, self.bt.get('bt_experimenters'))
         self.assertEqual(self.wavelength, self.bt.get('bt_wavelength'))
         self.assertEqual(os.getcwd(), self.home_dir)
-        # FIXME
-        # strtScnLst = ['bt_bt.yml','ex_l-user.yml','sa_l-user.yml','sp_ct_0.1.yml','sp_ct_0.5.yml','sp_ct_1.yml','sp_ct_5.yml','sp_ct_10.yml','sp_ct_30.yml']
-        # self.assertEqual(newobjlist,strtScnLst)
+        # test prepoluate ScanPlan
+        self.assertEqual(len(self.bt.scanplans), 3)
+        self.assertEqual(self.bt.scanplans[0]['sp_args'], (5,))
```

Comment:
@sbillinge functionality of ``ScanPlan`` objects are tested in ``test_beamtime.py`` (it is generated by exactly the same syntax and I test whether it can be created, updated when ``bt`` is changed, reloaded and run by RunEngine). Is there any suggested aspect I should cover?

for ``min`` argument, I need to implement it carefully as this might relate to the API of ``ScanPlan``.  Currently ``ct_0.1`` and ``ct_10``  in ``bt.list()`` is generated by inspecting function argument which is a float. I would prefer to separate it as another PR to ScanPlan as well.